gnocchi/rest/api.py

```diff
@@ -2055,6 +2057,7 @@ def __init__(self):
             "capabilities": CapabilityController(),
             "status": StatusController(),
             "aggregates": agg_api.AggregatesController(),
+            "influxdb": influxdb.InfluxDBController(),
```

Comment:
can we enable this on demand?

Reply:
Probably, though we don't do that for Prometheus either. So what would be the usecase?

Reply:
we do, it's only enabled if the right libs are there although we can't do the same here... i just don't want to load it unless explicit.

Reply:
If I have snappy installed on my system for whatever reason then I cannot disable Prometheus support.
This isn't a correct response. :/

That being said, we need something more than just a switch IMHO, see #499. So I'd postponed any switch until we solve #499.